src/Coverage.Report.Tests/Coverage.Report.Tests.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should we add a `.props` with versioning and common properties in this PR?

Reply:
Consider including stylecop checks.

Reply:
>Should we add a .props with versioning and common properties in this PR?

We can do it in PR in which we add second test project.

>Consider including stylecop checks.

May be in this PR or immediate next PR.